src/lib/mina_base/parties_logic.ml

```diff
@@ -557,9 +557,16 @@ module Make (Inputs : Inputs_intf) = struct
             ~else_:local_state.transaction_commitment
       }
     in
-    let update_local_excess = Bool.(is_start' ||| is_last_party) in
+    let update_local_excess =
+      is_last_party
```

Comment:
Revert this, the old version was the correct behaviour, and this is only masking the actual fee-excess issue. See comments/requested changes on #9887 for the correct way to fix that issue.

Reply:
This is handled in #9887.